CMakeLists.txt

```diff
@@ -118,7 +119,25 @@ foreach(SAMPLE ${SAMPLES})
     endif()
 
     add_dependencies(${SAMPLE_NAME} CopyZdf)
+
     if(WIN32)
         add_dependencies(${SAMPLE_NAME} CopyDlls)
     endif()
 endforeach()
+
+# TODO: Generalize how input file dependencies are copied, see issue #46
+    message(STATUS "Specialized operation for HandEyeCalibration")
```

Comment:
Final comment: I don't understand why this message is here. If those samples are disabled, it will still be printed, which is confusing. And even if they're enabled, I don't see that output adding any value. I suggest turning into a comment or just removing it.